hyperspy/_components/expression.py

```diff
@@ -162,6 +163,13 @@ def __init__(self, expression, name, position=None, module="numpy",
                  linear_parameter_list=None, check_parameter_linearity=True,
                  **kwargs):
 
+        if module is None:
+            try:
+                import numexpr
+                module = "numexpr"
+            except ImportError:
```

Comment:
Add logger entry to inform the user of the possibility to speed up things by installing `numexpr`?